### DISCUSSION TITLE
PluginServer for Kong

### DISCUSSION THREAD

**Discussion by RagunathInGIT** [Kong/kong#7625] [Help] [answered]
I have Kong running on K8s backed with Postgres DB. We have a requirement where need to build plugins in GOLang and JavaScript. Can someone please guide how to build pluginserver for these programming languages in the above mentioned setup.
Kong - 2.5.0 and KIC - 1.3.1

**Comment by guanlan** [comment, marked as answer]
Hi, 
Hope you find the following tutorials helpful:
https://konghq.com/blog/kong-gateway-plugin-javascript/
https://konghq.com/blog/kong-gateway-go-plugin
